src/components/Countdown/Countdown.tsx

```diff
@@ -13,6 +13,10 @@ export interface CountdownProps extends ComponentProps<'div'> {
    * The date the countdown should end
    */
   endDateTime: Date;
+  /**
+   * Function to modify strings coming from date-fns (Minutes -> Mins etc)
+   */
+  formatDurationStr?: (duration: string) => string;
```

Comment:
instead of passing the actual string, it might be better to mirror the way that `formatDuration` does it: https://date-fns.org/v4.1.0/docs/formatDuration.  `duration: number, interval: `day | minute ...`. This would allow having the default translations for these durations returned by `date-fns` based on locale, then allow the custom shorter translation where necessary, since the string replace would be different for each translation.

Reply:
This does allow for the strings to be returned from date-fns. This function are passed those strings via `durationParts` in the Duration component and then being returned with any changes the function wants to perform on them.